Sources/SwiftCurrent_UIKit/Extensions/WorkflowDecodableExtensions_UIKit.swift

```diff
@@ -0,0 +1,31 @@
+//  swiftlint:disable:this file_name
+//  WorkflowDecodableExtensions_UIKit.swift
+//  SwiftCurrent
+//
+//  Created by Tyler Thompson on 1/18/22.
+//  Copyright © 2022 WWT and Tyler Thompson. All rights reserved.
+//  
+
+import SwiftCurrent
+import UIKit
+
+extension WorkflowDecodable where Self: UIViewController & FlowRepresentable {
+    /// Decodes a ``LaunchStyle`` from a string.
```

Comment:
Double tick check

Reply:
What does this mean? That you don't think it should be? That it needs attention? Arguably, they should all be double tick, since Docc works better with that.

Reply:
Mostly just a "Did you mean to do this?" since others are single ticks

Reply:
That's fair. I'm not in a spot where I can easily change it, but calling out the inconsistency feels right. I'm cool with either a concerted effort to add two to everything, or to just remove from here.

Reply:
I can't get to a good spot to do this myself however, so if you've got the spare time, it could be resolved much faster.